### DISCUSSION TITLE
When run 'npx prisma generate' throw an error: Invoked Dart programs must have a 'main' function defined

### DISCUSSION THREAD

**Discussion by sdfgssdf** [medz/dart-orm#378] [Q&A] [answered]
<img width="492" alt="Screenshot 2024-06-03 at 8 00 15 AM" src="https://github.com/medz/prisma-dart/assets/20835278/49cbac1a-3ed4-4b5b-b54d-9571710fef0f">

**Comment by medz** [comment]
@sdfgssdf The problem you are experiencing does not occur at the moment. Can you give me a minimal error demo so that I can troubleshoot this problem?

**Reply by sdfgssdf** [reply, marked as answer]
I truly appreciate your response and it is now working.

I was following my existing template app and forgot to run dart pub get.
